lib/pure/mersenne.nim

```diff
@@ -35,14 +36,16 @@ proc newMersenneTwister*(seed: uint32): MersenneTwister =
                                       (result.mt[i-1] shr 30'u32)) + i)
 
 proc generateNumbers(m: var MersenneTwister) =
+
   for i in 0..623:
     var y = (m.mt[i] and 0x80000000'u32) +
             (m.mt[(i+1) mod 624] and 0x7fffffff'u32)
     m.mt[i] = m.mt[(i+397) mod 624] xor uint32(y shr 1'u32)
     if (y mod 2'u32) != 0:
       m.mt[i] = m.mt[i] xor 0x9908b0df'u32
 
-proc getNum*(m: var MersenneTwister): uint32 =
+proc getNum*(m: var MersenneTwister): uint32 {.deprecated:
+"deprecated, use random.rand() instead"} =
```

Comment:
```suggestion
"use random.rand() instead"} =
```
No need to repeat that it's deprecated, the warning will say that anyway (same for the other annotations).